src/config/locales/i18n.test.js

```diff
@@ -0,0 +1,65 @@
+import {
```

Comment:
I like having a static test for this, I was wondering if we should also modify the i18n library to throw a warning in the logs at least as the original story suggested since that maybe easier to handle the dynamic keys. I'm fine if that is tabled as solution for another time though.

Reply:
Are there additional steps to run this test, locally when I run `make` or `make test-react` it fails 

  FAIL  src/config/locales/i18n.test.js (21.975s)
  ● i18n › does not have missing strings

    expect(received).toBe(expected) // Object.is equality

    Expected: 0
    Received: 15

      45 |       console.log(report)
      46 |     }
    > 47 |     expect(report.length).toBe(0)
         |                           ^
      48 |   })
      49 |
      50 |   it('does not have unused strings', () => {

      at Object.<anonymous> (src/config/locales/i18n.test.js:47:27)

  ● i18n › does not have unused strings

    expect(received).toBe(expected) // Object.is equality

    Expected: 0
    Received: 2154

      55 |       console.log(report)
      56 |     }
    > 57 |     expect(report.length).toBe(0)
         |                           ^
      58 |   })
      59 |
      60 |   it.skip('does not have duplicate strings', () => {

      at Object.<anonymous> (src/config/locales/i18n.test.js:57:27)

Reply:
I did a couple force pushes after I messed up trying to squash some commits, could you try pulling down the latest again and seeing if you are still getting these failures? They should have been resolved already.

Reply:
I did have the missing strings being logged to the console, but I wasn't sure if that would jump out enough to make it useful, but I think it can't hurt. I'll add it back in, and maybe only log them on dev, does that sound good?

Reply:
I think only logging them in dev would be great, if it's not too much trouble. I'll pull the latest and confirm.

Reply:
Works on latest commits thanks!